src/uikit.cs

```diff
@@ -21262,6 +21274,10 @@ interface UITabBarItemAppearance : NSCopying, NSSecureCoding {
 	// 	[Export ("applySnapshot:animatingDifferences:")]
 	// 	void ApplySnapshot (NSDiffableDataSourceSnapshot<SectionIdentifierType, ItemIdentifierType> snapshot, bool animatingDifferences);
 
+	// 	[Async]
```

Comment:
Commented out for a reason?

Reply:
@chamons Yes, this needs some registrar changes, the binding will be enabled in a later PR.

Reply:
Issue please :)

Reply:
Already here :) https://github.com/xamarin/xamarin-macios/issues/6567

Reply:
@chamons it was part of the previous UIKit PR, just updated the commented out binding.

Reply:
Added to before we ship issue.